src/methods/fetch-authentication.js

```diff
@@ -0,0 +1,4 @@
+export default (self, session) => {
+  const url = `/authentications/${session.my_id}.json`
+  return self.apiRequest(url).then(resp => resp.data)
```

Comment:
provavelmente este método será chamado mais de uma vez e em locais diferentes, não tenho certeza se é uma boa solução (me parece uma gambiarra), mas pensei em também salvar no `session` e antes de enviar uma nova requisição na api verificar se já não está lá, o que você acha?

talvez com um `setTimeout` pra resetar esse objeto (deve ficar desatualizado) e um outro parâmetro do método para opcionalmente forçar a requisição mesmo que o `session.authentication` já esteja salvo

```suggestion
  return self.apiRequest(url).then(({ data }) => {
    session.authentication = data
    return data
  })
```